lrrbot/commands/card.py

```diff
@@ -6,6 +6,9 @@
 import common.time
 from common.cardname import clean_text
 
+CARD_GAME_MTG = 1
```

Comment:
This enum should be in a single place and not duplicated across five files. Maybe in `common.cardname`? Although it should then be called `common.card` or something instead.